play/src/front/Components/EmbedScreens/CoWebsitesContainer.svelte

```diff
@@ -1,53 +1,392 @@
+    import JitsiCowebsiteComponent from "../Cowebsites/JistiCowebsiteComponent.svelte";
+    import SimpleCowebsiteComponent from "../Cowebsites/SimpleCowebsiteComponent.svelte";
     import { JitsiCoWebsite } from "../../WebRtc/CoWebsite/JitsiCoWebsite";
-    import { log } from "console";
-    import { coWebsiteManager } from "../../WebRtc/CoWebsiteManager";
-    import { CoWebsite } from "../../WebRtc/CoWebsite/CoWebsite";
+    import { SimpleCoWebsite } from "../../WebRtc/CoWebsite/SimpleCoWebsite";
+    import { BBBCoWebsite } from "../../WebRtc/CoWebsite/BBBCoWebsite";
+    import BigBlueButtonCowebsiteComponent from "../Cowebsites/BigBlueButtonCowebsiteComponent.svelte";
+    import CoWebsiteTab from "./CoWebsiteTab.svelte";
+    import { max, min } from "lodash";
+    import { ArrowDownIcon } from "svelte-feather-icons";
 
-    export let vertical = false;
-    export let active = false;
-</script>
+    let activeCowebsite = $coWebsites[0].getId();
+    let showDropdown = false;
+    let tabBar: HTMLElement;
+    let showArrow = false;
+    let cowebsiteContainer = document.getElementById("cowebsites-container");
+    let container: HTMLElement;
+    let resizeBar: HTMLElement;
+    let startY: number;
+    let startX: number;
+    let startWidthContainer: number;
+    let widthContainer: number;
+    let startHeight: number;
+    const mediaQuery = window.matchMedia("(max-width: 768px)");
+    let totalTabsWidth = 0;
+    let initialWidth: number;
+    const widthTab = 300;
+    const widthTabResponsive = 195;
+    let vertical: boolean;
+
+    onMount(() => {
+        mediaQuery.addEventListener("change", (e: any) => handleTabletChange(e));
+        handleTabletChange(mediaQuery);
+    });
+
+    function handleTabletChange(e: MediaQueryList) {
+        if (e.matches) {
+            vertical = true;
+            resizeMobile();
+        } else {
+            vertical = false;
+            resizeDesktop();
+        }
+    }
+
+    function resizeDesktop() {
+        initialWidth = parseInt(getComputedStyle(container).width);
+        startWidthContainer = parseInt(getComputedStyle(container).width);
+
+        const handleMouseDown = (e: { clientX: number }) => {
+            startX = e.clientX;
+            startWidthContainer = parseInt(getComputedStyle(container).width);
+            document.addEventListener("mousemove", handleMouseMove);
+            document.addEventListener("mouseup", handleMouseUp);
+        };
+        resizeBar.addEventListener("mousedown", handleMouseDown);
+
+        const handleMouseMove = (e: { clientX: number }) => {
+            widthContainer = startWidthContainer - (e.clientX - startX);
+            container.style.width = widthContainer + "px";
+            initialWidth = parseInt(getComputedStyle(container).width);
+
+            const maxWidth = min([widthContainer, window.innerWidth - 350]);
+            container.style.width = maxWidth + "px";
+        };
```

Comment:
Maybe try to use a Svelte lib to implement the "draggable"? https://www.neodrag.dev/docs/svelte